dashboard/src/app/stacks/stack-details/stack.html

```diff
@@ -47,6 +47,28 @@
     </div>
   </che-label-container>
 
+  <!-- stack's tags -->
+  <che-label-container class="stack-tags-label"
+                       che-label-name="Tags"
+                       che-label-description="Tags define templates available for workspace">
+    <div layout="column" flex="100" class="stack-tags">
+      <md-chips flex name="stack's tags"
+                ng-model="stackController.stack.tags"
+                md-transform-chip="stackController.handleTagAdding($chip)"
+                md-on-add="stackController.updateEditorContent()"
+                md-on-remove="stackController.updateEditorContent()"
+                secondary-placeholder="Enter stack's tags">
+        <md-chip-template>
+          <div class="stack-library-text">{{$chip | uppercase}}</div>
+          <div class="stack-library-btn">
+            <i class="fa fa-close"></i>
+          </div>
+        </md-chip-template>
+      </md-chips>
+      <div class="stack-tags-reset" data-ng-click="stackController.stack.tags.length = 0">Reset</div>
```

Comment:
Make Reset button not 100% width of the row - now click below the input causes clearing the tags.